PotLock/activity-dao
# feed/interest slug updates and minor UI fixes.

## Summary
Updated feed/interest slug_ids to match farcaster channel data populated to feeds.
Updated loading animation to render at 60 fps, creating a smoother animation flow and adding more emojis for less white space.
Updated components naming convention.


## Related Issue
laggy home page loading animation


## Type of Change



## Testing Steps
npm run dev
load home page for loading animation fix
check feeds page for better-mapped farcaster channels
